libobs/obs-internal.h

```diff
@@ -284,15 +284,14 @@ struct obs_core_video_mix {
 	volatile bool gpu_encode_stop;
 
 	video_t *video;
+	struct obs_video_info ovi;
 
-	bool gpu_conversion;
```

Comment:
Does this have to be relocated? Moving this to ovi means writing to ovi, which does technically change behavior for `obs_get_video_info()`, returning the current gpu conversion state instead of previously the requested gpu conversion state.

Reply:
It doesn't have to be relocated, but thought it would be potentially nice to have if someone needs to query that state being turned off. I'll revert this to have less potential trouble

Reply:
Yeah, it's best to try to preserve behavior as much as possible with a change like this, since it complicates bug testing. If there's a need to get the gpu conversion state, then a new libobs function may be a better option.